distribution/src/main/resources/bin/elasticsearch

```diff
@@ -130,10 +130,30 @@ export HOSTNAME
 daemonized=`echo $* | grep -E -- '(^-d |-d$| -d |--daemonize$|--daemonize )'`
 if [ -z "$daemonized" ] ; then
     exec "$JAVA" $JAVA_OPTS $ES_JAVA_OPTS -Des.path.home="$ES_HOME" -cp "$ES_CLASSPATH" \
-          org.elasticsearch.bootstrap.Elasticsearch start "$@"
-else
-    exec "$JAVA" $JAVA_OPTS $ES_JAVA_OPTS -Des.path.home="$ES_HOME" -cp "$ES_CLASSPATH" \
-          org.elasticsearch.bootstrap.Elasticsearch start "$@" <&- &
+        org.elasticsearch.bootstrap.Elasticsearch start "$@"
+    exit $?
 fi
 
+pidfile=$(echo $* | awk '{match($0, /-p(idfile)? +([^ ]+)/,a);print a[2]}')
```

Comment:
Executing `bin/elasticsearch -d -p pid` produces the error `awk: line 1: syntax error at or near ,`